docs/01-eligibility.md

```diff
@@ -0,0 +1,48 @@
+# 1. Eligibility
+
+Protocol Guild eligible projects must:
+
+- Be fully open source: both “source available” and free to fork, modify, redistribute
+- Have a regular presence in R&D or governance venues, such as
+  - specification repos (e.g. `consensus-specs`, `execution-specs`, `execution-apis`)
+  - research forums (e.g. [ethresear.ch](https://ethresear.ch)), feature prototyping (e.g. EIPs, devnets, etc.)
+  - regular protocol calls (e.g. AllCoreDevs, testing/interop calls, etc.)
+- Target at least one of the following projects/areas:
+  - Ethereum core protocol maintenance and development
+    - Contributors to well-tested, technically-differentiated and production-ready Ethereum mainnet client implementations. Currently, this includes [Erigon](https://github.com/ledgerwatch/erigon), [EthereumJS](https://github.com/ethereumjs),  [Geth](https://github.com/ethereum/go-ethereum), [Hyperledger Besu](https://github.com/hyperledger/besu), [Lighthouse](https://github.com/sigp/lighthouse), [Lodestar](https://github.com/ChainSafe/lodestar), [Nethermind](https://github.com/NethermindEth/nethermind), [Nimbus](https://github.com/status-im/nimbus-eth2), [Prysm](https://github.com/prysmaticlabs/prysm), [Teku](https://github.com/ConsenSys/teku) and [Reth](https://github.com/paradigmxyz/reth)
+    - Client testing/security/infra which supports these implementations: [ethereum/tests](https://github.com/ethereum/tests) [ethereum/execution-spec-tests](https://github.com/ethereum/execution-spec-tests)
+    - Coordination related to upgrades and maintenance: [ethereum/pm](https://github.com/ethereum/pm)
+  - Research and implementation experiments related to potential protocol changes
+    - [Verkle tries](https://github.com/gballet/go-verkle) (Verge)
+    - [Portal Network](https://github.com/ethereum/portal-network-specs) (Purge)
+    - EVM improvements: [Ipsilon](https://github.com/ipsilon)
+    - Consensus work
+    - Cryptography
+    - Mechanism design
+    - Resource pricing
+  - Spec work resulting from the above (should be implementation agnostic, unopinionated)
+    - [Execution specs (EELS)](https://github.com/ethereum/execution-specs)
+    - [Consensus specs](https://github.com/ethereum/consensus-specs)
+  - Protocol Guild
+    - General comms
```

Comment:
I'd avoid abbreviations in docs, and perhaps move this at the bottom of the list?